bonita/content.md

```diff
@@ -14,397 +14,281 @@ Bonita (called Bonita BPM till 7.5) is an open-source business process managemen
-	$ docker exec -ti bonita_postgres /bin/bash
-	```
+`BIZ_DB_USER` default value is businessuser.
 
-	```console
-	$ cd /opt/bonita/BonitaCommunity-2021.2-u0/setup
-	$ ./setup.sh pull
-	$ TENANT_LOGIN=tech_user
-	$ TENANT_PASSWORD=secret
-	$ PLATFORM_LOGIN=pfadmin
-	$ PLATFORM_PASSWORD=pfsecret
-	$ sed -e 's/^#userName\s*=.*/'"userName=${TENANT_LOGIN}"'/' \
-	      -e 's/^#userPassword\s*=.*/'"userPassword=${TENANT_PASSWORD}"'/' \
-	      -i platform_conf/current/tenants/1/tenant_engine/bonita-tenant-community-custom.properties
-	$ sed -e 's/^platform.tenant.default.username\s*=.*/'"platform.tenant.default.username=${TENANT_LOGIN}"'/' \
-	      -e 's/^platform.tenant.default.password\s*=.*/'"platform.tenant.default.password=${TENANT_PASSWORD}"'/' \
-	      -i platform_conf/current/platform_portal/platform-tenant-config.properties
-	$ sed -e 's/^#platformAdminUsername\s*=.*/'"platformAdminUsername=${PLATFORM_LOGIN}"'/' \
-	      -e 's/^#platformAdminPassword\s*=.*/'"platformAdminPassword=${PLATFORM_PASSWORD}"'/' \
-	      -i platform_conf/current/platform_engine/bonita-platform-community-custom.properties
-	$ sed -i -e 's/^#GET|/GET|/' -e 's/^#POST|/POST|/' -e 's/^#PUT|/PUT|/' -e 's/^#DELETE|/DELETE|/' -i platform_conf/current/tenants/1/tenant_portal/dynamic-permissions-checks.properties
-	$ ./setup.sh push
-	```
+`BIZ_DB_PASS` default value is businesspass.
 
-	```console
-	$ docker restart bonita_postgres
-	```
 
--	Specific consideration regarding migration to Java 11 in Bonita 7.9
+## Logger configuration
 
-	Bonita 7.9 docker image runs with Java 11. If you are migrating from an earlier version which runs Java 8, you should validate on a test environment that your custom code is compatible. Aside from just code incompatibility, special attention has to be given to the dependencies of the custom code, as they might not work in Java 11.
+The logger can be configured by mounting a volume on folder `/opt/bonita/conf/logs` containing the configuration files.
 
-For more details regarding Bonita migration, see the [documentation](https://documentation.bonitasoft.com/bonita/2021.2/migrate-from-an-earlier-version-of-bonita-bpm).
+the volume must contain the 2 files
+[log4j2-loggers.xml](https://raw.githubusercontent.com/bonitasoft/bonita-distrib/7.14.0/tomcat-resources/tomcat-distrib-for-bonita/src/main/resources/tomcat/server/conf/log4j2-loggers.xml)
+and
+[log4j2-appenders.xml](https://raw.githubusercontent.com/bonitasoft/bonita-distrib/7.14.0/docker/files/log4j2/log4j2-appenders.xml)
 
 ## Security
 
 This Docker image activates both static and dynamic authorization checks by default on REST API. To be consistent, it also deactivates the HTTP API.
 
 -	REST API authorization
 
-	-	[Static authorization checking](https://documentation.bonitasoft.com/bonita/2021.2/rest-api-authorization#static_authorization)
+	-	[Static authorization checking](https://documentation.bonitasoft.com/bonita/2022.1/rest-api-authorization#static_authorization)
```

Comment:
I would use latest everywhere, this is easier for later maintenance of this page...
```suggestion
	-	[Static authorization checking](https://documentation.bonitasoft.com/bonita/latest/rest-api-authorization#static_authorization)
```